src-java/flowhs-topology/flowhs-storm-topology/src/test/java/org/openkilda/wfm/topology/flowhs/mapper/HaFlowMapperTest.java

```diff
@@ -155,7 +169,40 @@ null, new FlowEndpoint(SWITCH_ID_1, PORT_1, VLAN_1, INNER_VLAN_1), FlowStatus.UP
         assertSubFlow(subFlow, SUB_FLOW_2_NAME, result);
     }
 
-    private static void assertSubFlows(Set<HaSubFlow> expectedList, List<HaSubFlowDto> actualSet) {
+    @Test
+    public void toHaSubFlowTest() {
+        HaSubFlowDto subFlow = new HaSubFlowDto(
+                SUB_FLOW_1_NAME, new FlowEndpoint(SWITCH_ID_1, PORT_1, VLAN_1, INNER_VLAN_1), FlowStatus.UP, DESC_2,
+                Instant.MIN, Instant.MAX);
+        HaSubFlow result = mapper.toSubFlow(subFlow);
+        assertSubFlow(subFlow, subFlow.getFlowId(), result);
+    }
+
+    @Test
+    public void toRequestedFlowsTest() {
+        List<HaSubFlowDto> subFlows = Lists.newArrayList(SUB_FLOW_1, SUB_FLOW_2);
+        HaFlowRequest request = new HaFlowRequest(
+                HA_FLOW_ID, SHARED_ENDPOINT, BANDWIDTH, PathComputationStrategy.COST, FlowEncapsulationType.VXLAN,
+                MAX_LATENCY, MAX_LATENCY_TIER_2, true, false, true, PRIORITY, false, DESC_1, true, FLOW_3,
+                subFlows, Type.CREATE);
+        request.setSubFlows(Lists.newArrayList(SUB_FLOW_1, SUB_FLOW_2));
+
+        Collection<RequestedFlow> requestedFlows = mapper.toRequestedFlows(request);
+        assertEquals(2, requestedFlows.size());
+        Map<String, RequestedFlow> requestedFlowMap = requestedFlows.stream()
+                .collect(Collectors.toMap(RequestedFlow::getFlowId, Function.identity()));
+        assertEquals(2, requestedFlowMap.size());
```

Comment:
these 2 assertions allows the requested flows to be any flows. I suggest to use an assertion that there are exactly those flows from 183 with ids `SUB_FLOW_1, SUB_FLOW_2`